File: articles/proxy-variables-and-measurement-error/fortran/proxy_measurement_risk.f90
program proxy_measurement_risk
  implicit none
  real :: validity_gap, missingness, differential_error, label_error, score
  validity_gap = 0.42
  missingness = 0.12
  differential_error = 0.24
  label_error = 0.08
  score = (validity_gap + missingness + differential_error + label_error) / 4.0
  print *, "measurement_risk_score=", score
end program proxy_measurement_risk

File: articles/proxy-variables-and-measurement-error/java/ProxyMeasurementRisk.java
public class ProxyMeasurementRisk {
    public static void main(String[] args) {
        double validityGap = 0.42;
        double missingness = 0.12;
        double differentialError = 0.24;
        double labelError = 0.08;
        double score = (validityGap + missingness + differentialError + labelError) / 4.0;
        System.out.printf("measurement_risk_score=%.4f%n", score);
    }
}
